### PR TITLE
Explain how to do time based bash watch

### PR DESCRIPTION
**Pull request by SUPERCILEX**

### CODE REVIEW

File: crates/nu-command/src/filesystem/watch.rs [hunk 1]
```diff
@@ -282,6 +282,11 @@ impl Command for Watch {
                 example: r#"watch /foo/bar { |op, path| $"($op) - ($path)(char nl)" | save --append changes_in_bar.log }"#,
                 result: None,
             },
+            Example {
+                description: "Note: if you are looking to run a command every N units of time, this can be accomplished with a loop and sleep",
+                example: r#"loop { command; sleep time }"#,
```
Comment (fdncred) [inline crates/nu-command/src/filesystem/watch.rs:287]: seems reasonable. we call these units of time durations
```suggestion
                example: r#"loop { command; sleep duration}"#,
```

Comment (SUPERCILEX) [reply crates/nu-command/src/filesystem/watch.rs]: Done

**Comment by fdncred** [comment]
@rgwood Since (I believe) you wrote `watch` I wanted to give you the opportunity to object / land this PR. I could probably live with it, but something about it strikes me odd of putting an example for another command in it.

**Comment by rgwood** [comment]
Bit weird to me but I don’t have an especially strong feeling about it. Might be useful for people looking for the unix functionality.

**Comment by fdncred** [comment]
Thanks @SUPERCILEX

**Pull request merged**